test/models/team_2_test.rb

```diff
@@ -1590,4 +1590,13 @@ def setup
     t = create_team
     assert_equal [], t.search_for_similar_articles('Test')
   end
+
+  test "should return platforms for which statistics are available" do
+    t = create_team
+    pm = create_project_media team: t
+    assert_equal [], t.statistics_platforms
+    create_tipline_request team_id: t.id, platform: 'telegram', associated: pm
+    create_tipline_request team_id: t.id, platform: 'whatsapp', associated: pm
```

Comment:
I suggest adding another TiplineRequest with the same platform so we can verify the group by platform.

Reply:
Thanks Sawy, good point - done!